src/kp_derivable.ml

```diff
@@ -71,24 +71,37 @@ end
 
 module Asn_OID = struct
   type t = Asn.OID.t
-  let show = Asn.OID.to_string
-  let pp = pp_of_to_string show
-  let compare a b =
-    String.compare (show a) (show b)
-  let equal a b = compare a b = 0
+  [@@deriving eq,ord,show]
+
+  let show oid =
```

Comment:
Why not sticking to the derived version ?

Reply:
Indeed, that works. Fixed!